src/ansys/mapdl/core/mapdl_geometry.py

```diff
@@ -907,3 +1479,77 @@ def _select_items(self, items, item_type, sel_type):
             self._mapdl.vsel(sel_type, vmin=items, return_mapdl_output=True)
         else:
             raise ValueError(f'Unable to select "{item_type}"')
+
+
+class LegacyGeometry(Geometry):
+    """Legacy pythonic representation of MAPDL CAD geometry
+
+    Contains advanced methods to extend geometry building and
+    selection within MAPDL.
+    """
+
+    def keypoints(self) -> np.array:  # type: ignore
+        """Keypoint coordinates"""
+        return super().get_keypoints(return_as_array=True)
+
+    def lines(self) -> pv.PolyData:
+        """Active lines as a pyvista.PolyData"""
+        return super().get_lines()  # type: ignore
+
+    def areas(self, quality=1, merge=False) -> pv.PolyData:
```

Comment:
The return type seems to be Union[pv.PolyData, list[UnstructuredGrid]]
```suggestion
    def areas(self, quality=1, merge=False) -> Union[pv.PolyData, list[UnstructuredGrid]]:
```